static/js/src/ua-payment-modal.js

```diff
@@ -746,7 +746,8 @@ function handleGuestPaymentMethodResponse(data) {
   ensurePurchaseAccount(
     customerInfo.email,
     customerInfo.accountName,
-    paymentMethod.id
+    paymentMethod.id,
+    customerInfo.address.country
```

Comment:
I'm refactoring this file at the moment but I'll keep this requirement in mind. Thanks!